### PR TITLE
Update docs with run script

### PR DESCRIPTION
**Pull request by pradavc**

### CODE REVIEW

File: docs/testnet/install/manual/configure.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ hide_table_of_contents: false
 
 # Configure ports and run
```
Comment (micho) [inline docs/testnet/install/manual/configure.md:6]: :::caution Important Security Notice
Docker exposes ports of your machine to the public. Please be aware that this setup does not implement any security measures. Running this testnet in an unprotected environment may expose your systems and data to potential risks. It is strongly recommended to deploy Diva in an isolated, secure and controlled network environment.
:::

File: docs/testnet/intro.md [hunk 2]
```diff
@@ -4,14 +4,22 @@ description: This introduction details the process of how to participate in the
 hide_table_of_contents: false
 ---
 
-<div style={{textAlign: 'center'}}>
-    <img src="https://diva.community/metalogo.png"  width="200"/>
-</div>
+:::caution Important Security Notice
+This testnet runs on Docker containers, which may expose your machine's ports to the public. Please be aware that this setup does not implement any security measures. Running this testnet in an unprotected environment may expose your systems and data to potential risks. Deploying Diva in a secure and controlled network environment is strongly recommended.
+:::
```
Comment (micho) [inline docs/testnet/intro.md:9]: this is here too. is it worth having it twice? imo yes as its a security notice. but i dont feel strongly about it

File: docs/testnet/update-diva.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Update Diva
+hide_table_of_contents: false
+---
```
Comment (micho) [inline docs/testnet/update-diva.md:4]: nice to have the Update Diva page!

File: docs/testnet/install/manual/prepare/own-clients.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+### COMPOSE_FILE
+
+The Diva Testnet repository comes with a few examples of docker compose files.
+
+Replace the value `docker-compose.yml` of the `COMPOSE_FILE` with the name of the docker compose file that you want to use).
+
+```bash
+COMPOSE_FILE=docker-compose.yml # Change this (docker compose file name)
+```
```
Comment (micho) [inline docs/testnet/install/manual/prepare/own-clients.md:42]: there was a previous comment from @bertho-zero asking if we could explain the different docker compose files. how do you feel about it @pradavc?

File: docs/testnet/install/manual/configure.md (hunk 1 above)
Comment (micho) [reply docs/testnet/install/manual/configure.md:6]: i know this is below, but is it worth repeating here, as its a security warning?

**Review by micho — COMMENTED** [review]
Thanks for updating this!! Thew new docs feel much nicer, and love the screenshots.

Leaving minor comments

File: docs/testnet/install/manual/prepare/own-clients.md (hunk 4 above)
Comment (bertho-zero) [reply docs/testnet/install/manual/prepare/own-clients.md:42]: Everything is in the `docker-compose.yml` now, to use clients you have to add the docker profile `clients`, for metrics the profile `metrics` and `telemetry` for telemetry:

```bash
DOCKER_PROFILES=clients,metrics,telemetry
```

**Pull request merged**